depmap/nbconverted/1.explore_depmap.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Describe Cancer Dependency Map Data
+
+# In[1]:
+
+
+import pathlib
+import pandas as pd
```

Comment:
Consider removing these imports if they aren't needed.

Reply:
yes! thank you for pointing out